drivers/volume/portworx/portworx.go

```diff
@@ -1265,18 +1265,24 @@ func (d *portworx) RejoinNode(n node.Node) error {
 			Cause: err.Error(),
 		}
 	}
-	if err := k8s.Instance().RemoveLabelOnNode(n.Name, "px/service"); err != nil {
+	if err = k8s.Instance().RemoveLabelOnNode(n.Name, "px/service"); err != nil {
```

Comment:
use cont for `"px/service"`. It's already defined in drivers/volume/portworx/schedops/k8s-schedops.go. Export it.

Same for px/enabled.